assume_role.sh

```diff
@@ -1,13 +1,15 @@
+#!/bin/bash
```

Comment:
if you wouldn't  mind removing the shebang also as it's a little misleading, the script if run with just bash and no source has no purpose.

Reply:
👍